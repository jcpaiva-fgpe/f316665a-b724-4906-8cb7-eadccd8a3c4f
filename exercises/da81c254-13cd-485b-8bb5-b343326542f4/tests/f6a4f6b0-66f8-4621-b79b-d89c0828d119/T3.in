22 47
23 57
