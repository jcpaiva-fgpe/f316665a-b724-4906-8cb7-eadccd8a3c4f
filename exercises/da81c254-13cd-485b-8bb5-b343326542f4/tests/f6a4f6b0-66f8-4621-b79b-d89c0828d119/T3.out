Passaram apenas 70 minutos!
Queres dizer, 1 hora e 10 minutos?!
